docs/ad/index.md

```diff
@@ -25,11 +23,14 @@ To get started, choose **Anomaly Detection** in Kibana.
 
 A detector is an individual anomaly detection task. You can create multiple detectors, and all the detectors can run simultaneously, with each analyzing data from different sources.
 
-1. Choose **Create Detector**.
-2. For **Define detector**, enter the **Name of the detector** and a brief **Description**. Make sure the name that you enter is unique and descriptive enough to help you to identify the purpose of this detector.
-3. For **Data source**, choose the index that you want to use as the data source. You can optionally use index patterns to choose multiple indices. Choose the **Timestamp field** in your index.
-4. For **Filter Data**, you can optionally filter the index that you chose as the data source. From the **Filter type** menu, choose  **Visual filter**, and then design your filter query by selecting **Fields**, **Operator**, and **Value**, or choose **Custom Expression** and add in your own JSON filter query.
-5. Choose **Create**.
+1. Choose **Detector**, **Create Detector**.
+1. Enter the **Name** of the detector and a brief **Description**. Make sure the name that you enter is unique and descriptive enough to help you to identify the purpose of this detector.
+1. For **Data source**, choose the index that you want to use as the data source. You can optionally use index patterns to choose multiple indices.
+1. Choose the **Timestamp field** in your index.
+1. For **Data filter**, you can optionally filter the index that you chose as the data source. From the **Filter type** menu, choose  **Visual filter**, and then design your filter query by selecting **Fields**, **Operator**, and **Value**, or choose **Custom Expression** and add in your own JSON filter query.
+1. For **Detector operation settings**, define the **Detector interval** to set the time interval at which the detector collects data. The shorter you set this interval, the more data points the detector gets, but the more computing resources it consumes.
+1. To add extra processing time for data collection, specify a **Window delay** value. Window delay is like a buffer that tells the detector to wait for a certain amount of time before processing the data. This is to account for any time your data source may need for internal processing.
```

Comment:
Window delay tells the detector to wait for a certain amount of time before processing the data. It can help account for any time your data source may need for internal processing.

Reply:
Done.